src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -774,10 +774,14 @@ g.test('non_zero_array_stride_and_attribute_offset')
     const arrayStride = t.makeLimitVariant('maxVertexBufferArrayStride', arrayStrideVariant);
     const formatInfo = kVertexFormatInfo[format];
     const formatSize = formatInfo.byteSize;
-    const offset = clamp(makeValueTestVariant(arrayStride, offsetVariant), {
-      min: 0,
-      max: arrayStride - formatSize,
```

Comment:
This clamped on Chrome for uint8x4 where arrayStride = 2048, multiplier = 0.4, add = -4:
(2048/2 - 4) is divisible by 4.

But fails on Safari where arrayStride = 204865532, multiplier = 0.4, add = -4
(65532/2 - 4) is not divisible by 4